src/modules/services/components/bulk/AssignServiceButton.tsx

```diff
@@ -77,23 +103,66 @@ const AssignServiceButton: React.FC<Props> = ({
     return 'Enroll + Assign';
   }, [client.activeEnrollment, isAssignedOnDate, localDisabled]);
 
+  const startIcon =
+    isAssignedOnDate && !localDisabled ? (
+      <CheckIcon />
+    ) : clientAlerts.length > 0 ? (
+      <WarningAmberRoundedIcon />
+    ) : undefined;
+
   return (
     <>
       <ButtonTooltipContainer title={disabledReason} placement='top-start'>
         <LoadingButton
           onClick={onClick}
           loading={loading}
           disabled={disabled || localDisabled}
-          startIcon={
-            isAssignedOnDate && !localDisabled ? <CheckIcon /> : undefined
-          }
+          startIcon={startIcon}
           fullWidth
           variant={isAssignedOnDate ? 'contained' : 'gray'}
         >
           {buttonText}
         </LoadingButton>
       </ButtonTooltipContainer>
       {apolloError && <ApolloErrorAlert error={apolloError} />}
+      <CommonDialog fullWidth open={clientAlertDialogOpen}>
+        <DialogTitle>
+          {clientAlerts.length === 1 ? 'Client Alert' : 'Client Alerts'}
```

Comment:
Left feedback in Figma. I feel like the client name should be here, it's not obvious who the alert is for.